modules/admin_manual/pages/installation/ubuntu_18_04.adoc

```diff
@@ -157,6 +157,12 @@ occ config:system:set trusted_domains 1 --value="$myip"
 
 === Set Up a Cron Job
 
+Set your backgroud job mode to cron
+
+----
+occ background:cron
```

Comment:
NO-NO-NO !!
Use:
```
[source,console,subs="attributes+"]
----
{occ-command-example-prefix} background:cron
----
```

Reply:
Why dow we need the `{occ-command-example-prefix}` 

Above in line 30, we create an occ that helps with all commands

Reply:
PR #2835 puts in the `[source,console,subs="attributes+"]` format stuff.

The actual `occ` command is OK, because this document has special instructions "Create the occ Helper Script" which tell you to create an `occ` script that "just works". Refactoring that would be a separate discussion...